lib/charms/opensearch/v0/helper_cluster.py

```diff
@@ -41,7 +43,7 @@ def generated_roles() -> List[str]:
 
     @staticmethod
     def get_cluster_settings(
-        opensearch: OpenSearchDistribution,
+        opensearch: "OpenSearchDistribution",
```

Comment:
@reneradoi, I've noticed that, if I run `tests/unit/lib/test_helper_cluster.py` unit test separated, it would fail because of a circular dependency. If I run it using the entire unit tests we have, then it works, likely because OpenSearchDistribution was already loaded by some other test.

I changed this so I could remove the dependency on "OpenSearchDistribution", which was causing this loop dependency in the first place!

Reply:
Check it here: https://pastebin.ubuntu.com/p/87F6qYwQDs/

Reply:
Ah I remember that, we briefly chatted about it. Thanks for the reminder!

Reply:
Guys, PLS let's not groom and grow circular dependencies in our code, as much as avoidable...

This problem can be trivially resolved using an application constant, instead of the hard-coded list of default roles in this module. See https://github.com/canonical/opensearch-operator/commit/d227aa9c48e540edc6b86a492cd15ae9962833e6